orienteer-core/src/test/java/org/orienteer/core/component/table/filter/OrientDbFilterTest.java

```diff
@@ -0,0 +1,278 @@
+package org.orienteer.core.component.table.filter;
+
+import com.google.common.collect.Lists;
+import com.orientechnologies.orient.core.db.ODatabase;
+import com.orientechnologies.orient.core.db.document.ODatabaseDocument;
+import com.orientechnologies.orient.core.metadata.schema.OClass;
+import com.orientechnologies.orient.core.metadata.schema.OProperty;
+import com.orientechnologies.orient.core.metadata.schema.OType;
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.orienteer.junit.GuiceRule;
+import org.orienteer.junit.StaticInjectorProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.ydn.wicket.wicketorientdb.utils.DBClosure;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Vitaliy Gonchar
+ */
+public class OrientDbFilterTest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OrientDbFilterTest.class);
```

Comment:
Please do the following with unnecessary LOG.info/LOG.error calls below: either transform to "assert" or just simply remove.